schemas/defs.schema.json

```diff
@@ -44,6 +44,10 @@
           },
           "type": "array"
         },
+        "description": {
+          "description": "Short description",
```

Comment:
What should we say about the Markdown syntax inside these strings? Since BCD has HTML in notes, my suggestion is to do the same: https://github.com/web-platform-dx/web-features/pull/882